app/views/jobseekers/job_applications/qualifications/select_category.html.slim

```diff
@@ -8,7 +8,14 @@
     = form_for form, url: submit_category_jobseekers_job_application_qualifications_path(job_application), method: :post do |f|
       = f.govuk_error_summary
 
-      = f.govuk_collection_radio_buttons :category, Qualification.categories.keys, :to_s
+      = f.govuk_radio_buttons_fieldset(:category, legend: { size: "s" }) do
+        = f.govuk_radio_button :category, :gcse, link_errors: true
+        = f.govuk_radio_button :category, :as_level
+        = f.govuk_radio_button :category, :a_level
+        = f.govuk_radio_button :category, :other_secondary
+        = f.govuk_radio_button :category, :undergraduate
+        = f.govuk_radio_button :category, :postgraduate, hint: { text: t(".hints.qualification_categories.postgraduate") }
```

Comment:
This looks a bit weird - can it not just have a hint in the standard mechanism like all the other buttons?

Reply:
Not 100% sure what you mean by this one sorry @starswan. The other radio buttons don't have hint text, it's just the post grad option (and I believe the other category will have hint text added on another ticket)